### ISSUE TITLE
Add `Make property required` codefix for `CS8618`

### ISSUE THREAD

**Issue by DoctorKrolic** [dotnet/roslyn#66089]
Consider the following code:
```cs
#nullable enable

class MyClass
{
    public string MyProperty { get; set; }
}
```

Here nullability warning `CS8618` is reported on `MyProperty`, because there is no guarantee that it isn't null after creating instance of `MyClass`. Currently there are 2 codefixes awailable: either make property nullable of create and assign it from the constructor. In .NET 7 and C# 11 there is a new way of preventing this warning to happen - make that property required. So I suggest to add a `Make property required` codefix when language version is 11 or greater and `RequiredAttribute` is present in the metadata.
